packages/components/src/data-model/word-query/word-query-resolver.js

```diff
@@ -52,6 +52,11 @@ export default class WordQueryResolver {
     if (!storage.words.has(key)) {
       // eslint-disable-next-line prefer-const
       let response = new WordQueryResponse({ updateCallback: this.updateCallback.bind(this, storage, key) })
+      // This makes an Apollo reactive variable
+      const word = makeVar(response.toJsonObject())
+      // Store the result object reference into the map
+      storage.words.set(key, word)
+
```

Comment:
`wordQuery.start()` on line 97 calls `notify()` which expects the `word` object to be in the storage. So it's better to place the `word` to the storage before the `wordQuery.start()` call.